src/components/top-navbar/navbar.scss

```diff
@@ -3,24 +3,34 @@
     display: flex;
     justify-content: space-around;
     align-items: center;
-    padding: 0.5rem 2rem;
+    padding: 1.4rem 1.4rem;
     background-color: #5d98f1;
```

Comment:
The background color needs to be removed. See the new design.

Reply:
done